diff -ruNp a/common.c b/common.c
--- a/common.c	2019-01-26 04:44:00.154513032 +0900
+++ b/common.c	2019-01-26 04:38:58.768099120 +0900
@@ -391,7 +391,7 @@ void read_sar_cpu_as_paragraph ( const c
         for ( i--; ( i > 0 ) && isspace ( line [ i -1 ] ) ; i-- )
             line [ i -1 ] = '\0';
         /* count cpu all line and set numbers to cpu_as_paragraph */
-        if ( strstr ( *(&line), "Average" ) != 0 )
+        if ( ( strstr ( *(&line), "Average" ) != 0 ) || ( strstr ( *(&line), "平均値" ) != 0 ) ) 
             break;
         if ( ( strstr ( *(&line), "all" ) != 0 ) ) 
             _cpu_as_paragraph++;
@@ -756,37 +756,37 @@ void sanity_check_file ( char * line, co
     if ( strstr ( line, "LINUX RESTART" ) != 0 )
         initialize_avg_checked_int ( );
 
-    if ( ( cpu == 1 ) && ( proc_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( cpu == 1 ) && ( proc_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         cpu_avg_checked_int = 1;
         cpu_avg_checked = "Checked";
     }
-    if ( ( proc_s == 1 ) && ( pswpin_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( proc_s == 1 ) && ( pswpin_s == 0 ) && ( ( !strstr ( line, "Average" ) ) || ( !strstr ( line, "平均値" ) ) ) )
     {
         proc_s_avg_checked_int = 1;
         proc_s_avg_checked = "Checked";
     }
-    if ( ( pswpin_s == 1 ) && ( pgpgin_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( pswpin_s == 1 ) && ( pgpgin_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         pswpin_s_avg_checked_int = 1;
         pswpin_s_avg_checked = "Checked";
     }
-    if ( ( pgpgin_s == 1 ) && ( tps == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( pgpgin_s == 1 ) && ( tps == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         pgpgin_s_avg_checked_int = 1;
         pgpgin_s_avg_checked = "Checked";
     }
-    if ( ( tps == 1 ) && ( kbmemfree == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( tps == 1 ) && ( kbmemfree == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         tps_avg_checked_int = 1;
         tps_avg_checked = "Checked";
     }
-    if ( ( kbmemfree == 1 ) && ( kbswpfree == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( kbmemfree == 1 ) && ( kbswpfree == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         kbmemfree_avg_checked_int = 1;
         kbmemfree_avg_checked = "Checked";
     }
-    if ( ( kbswpfree == 1 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( kbswpfree == 1 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         if ( ( kbhugfree == 1 ) || ( dentunusd == 0 ) )
         {
@@ -796,47 +796,47 @@ void sanity_check_file ( char * line, co
             kbhugfree_avg_checked = "Checked";
         }
     }
-    if ( ( kbhugfree == 1 ) && ( dentunusd == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( kbhugfree == 1 ) && ( dentunusd == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         kbhugfree_avg_checked_int = 1;
         kbhugfree_avg_checked = "Checked";
     }
-    if ( ( dentunusd == 1 ) && ( runq_sz == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( dentunusd == 1 ) && ( runq_sz == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         dentunusd_avg_checked_int = 1;
         dentunusd_avg_checked = "Checked";
     }
-    if ( ( runq_sz == 1 ) && ( dev == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( runq_sz == 1 ) && ( dev == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         runq_sz_avg_checked_int = 1;
         runq_sz_avg_checked = "Checked";
     }
-    if ( ( dev == 1 ) && ( rxpck_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( dev == 1 ) && ( rxpck_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         dev_avg_checked_int = 1;
         dev_avg_checked = "Checked";
     }
-    if ( ( rxpck_s == 1 ) && ( rxerr_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( rxpck_s == 1 ) && ( rxerr_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         rxpck_s_avg_checked_int = 1;
         rxpck_s_avg_checked = "Checked";
     }
-    if ( ( rxerr_s == 1 ) && ( call_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( rxerr_s == 1 ) && ( call_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         rxerr_s_avg_checked_int = 1;
         rxerr_s_avg_checked = "Checked";
     }
-    if ( ( call_s == 1 ) && ( scall_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( call_s == 1 ) && ( scall_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         call_s_avg_checked_int = 1;
         call_s_avg_checked = "Checked";
     }
-    if ( ( scall_s == 1 ) && ( totsck == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( scall_s == 1 ) && ( totsck == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         scall_s_avg_checked_int = 1;
         scall_s_avg_checked = "Checked";
     }
-    if ( ( totsck == 1 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( totsck == 1 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         if ( squeezd == 0 )
         {
@@ -846,7 +846,7 @@ void sanity_check_file ( char * line, co
             squeezd_avg_checked = "Checked";
         }
     }
-    if ( ( squeezd == 1 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( squeezd == 1 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         squeezd_avg_checked_int = 1;
         squeezd_avg_checked = "Checked";
diff -ruNp a/sar_analyzer.c b/sar_analyzer.c
--- a/sar_analyzer.c	2019-01-26 05:00:23.164294623 +0900
+++ b/sar_analyzer.c	2019-01-26 04:39:20.463697000 +0900
@@ -266,7 +266,7 @@ int set_token_column ( int file_number,
         /* if AM or PM will appear, column number should be decremented, unless, segfault in set_column_item() */
         if ( SAR_OPTION != 'Z' )
         {
-            if ( ( strstr ( token, "AM" ) != NULL ) || ( strstr ( token, "PM" ) != NULL ) )
+            if ( ( strstr ( token, "AM" ) != NULL ) || ( strstr ( token, "PM" ) != NULL ) || ( strstr ( token, "午前" ) != NULL ) || ( strstr ( token, "午後" ) != NULL ) ) 
                 i--;
         }
         /* this item name is set by get_word_line() */
@@ -746,12 +746,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_usr [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -847,12 +848,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_sys [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -933,12 +935,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_iowait [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -1019,12 +1022,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_idle [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -1121,12 +1125,13 @@ int set_token_items ( int file_number, c
                         set_tasks_lowest_date ( this_date_all, "proc" );
                         set_tasks_lowest_time ( time_value, "proc" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_tasks_proc [ file_number ] );
@@ -1217,12 +1222,13 @@ int set_token_items ( int file_number, c
                         set_tasks_lowest_date ( this_date_all, "cswch" );
                         set_tasks_lowest_time ( time_value, "cswch" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_tasks_cswch [ file_number ] );
@@ -1319,12 +1325,13 @@ int set_token_items ( int file_number, c
                         set_pswap_lowest_date ( this_date_all, "pswpin" );
                         set_pswap_lowest_time ( time_value, "pswpin" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_swapping_pswpin [ file_number ] );
@@ -1415,12 +1422,13 @@ int set_token_items ( int file_number, c
                         set_pswap_lowest_date ( this_date_all, "pswpout" );
                         set_pswap_lowest_time ( time_value, "pswpout" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_swapping_pswpout [ file_number ] );
@@ -1538,12 +1546,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "pgpgin" );
                         set_paging_lowest_time ( time_value, "pgpgin" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_pgpgin [ file_number ] );
@@ -1634,12 +1643,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "pgpgout" );
                         set_paging_lowest_time ( time_value, "pgpgout" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_pgpgout [ file_number ] );
@@ -1718,12 +1728,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "fault" );
                         set_paging_lowest_time ( time_value, "fault" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_fault [ file_number ] );
@@ -1801,12 +1812,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "majflt" );
                         set_paging_lowest_time ( time_value, "majflt" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_mjflt [ file_number ] );
@@ -1884,12 +1896,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "vmeff" );
                         set_paging_lowest_time ( time_value, "vmeff" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_vmeff [ file_number ] );
@@ -1994,12 +2007,13 @@ int set_token_items ( int file_number, c
                         set_io_transfer_rate_lowest_date ( this_date_all, "tps" );
                         set_io_transfer_rate_lowest_time ( time_value, "tps" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_io_transfer_rate_tps [ file_number ] );
@@ -2090,12 +2104,13 @@ int set_token_items ( int file_number, c
                         set_io_transfer_rate_lowest_date ( this_date_all, "bread" );
                         set_io_transfer_rate_lowest_time ( time_value, "bread" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_io_transfer_rate_bread [ file_number ] );
@@ -2173,12 +2188,13 @@ int set_token_items ( int file_number, c
                         set_io_transfer_rate_lowest_date ( this_date_all, "bwrtn" );
                         set_io_transfer_rate_lowest_time ( time_value, "bwrtn" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_io_transfer_rate_bwrtn [ file_number ] );
@@ -2283,12 +2299,13 @@ int set_token_items ( int file_number, c
                         set_memory_lowest_date ( this_date_all, "memused" );
                         set_memory_lowest_time ( time_value, "memused" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_memory_memused [ file_number ] );
@@ -2379,12 +2396,13 @@ int set_token_items ( int file_number, c
                         set_memory_lowest_date ( this_date_all, "kbcommit" );
                         set_memory_lowest_time ( time_value, "kbcommit" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_memory_kbcommit [ file_number ] );
@@ -2462,12 +2480,13 @@ int set_token_items ( int file_number, c
                         set_memory_lowest_date ( this_date_all, "commit" );
                         set_memory_lowest_time ( time_value, "commit" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_memory_commit [ file_number ] );
@@ -2647,12 +2666,13 @@ int set_token_items ( int file_number, c
                         set_kernel_table_lowest_date ( this_date_all, "dentunusd" );
                         set_kernel_table_lowest_time ( time_value, "dentunusd" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_kernel_table_dentunusd [ file_number ] );
@@ -2743,12 +2763,13 @@ int set_token_items ( int file_number, c
                         set_kernel_table_lowest_date ( this_date_all, "file" );
                         set_kernel_table_lowest_time ( time_value, "file" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_kernel_table_file [ file_number ] );
@@ -2826,12 +2847,13 @@ int set_token_items ( int file_number, c
                         set_kernel_table_lowest_date ( this_date_all, "inode" );
                         set_kernel_table_lowest_time ( time_value, "inode" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_kernel_table_inode [ file_number ] );
@@ -2951,12 +2973,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "runq_sz" );
                         set_ldavg_lowest_time ( time_value, "runq_sz" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_runq [ file_number ] );
@@ -3048,12 +3071,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "plist_sz" );
                         set_ldavg_lowest_time ( time_value, "plist_az" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_plist [ file_number ] );
@@ -3132,12 +3156,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "ldavg_one" );
                         set_ldavg_lowest_time ( time_value, "ldavg_one" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_ldavg_one [ file_number ] );
@@ -3215,12 +3240,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date( this_date_all, "ldavg_five" );
                         set_ldavg_lowest_time( time_value, "ldavg_five" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_ldavg_five [ file_number ] );
@@ -3298,12 +3324,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "ldavg_15" );
                         set_ldavg_lowest_time ( time_value, "ldavg_15" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_ldavg_15 [ file_number ] );
@@ -4757,7 +4784,7 @@ int get_word_line ( int file_number, cha
             initialize_check_int ( );
         }
         /* first, we are setting column number of interests */
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && (strstr ( *line, "usr" ) || strstr ( *line, "sys" )   != NULL ) )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) )  && ( ( strstr ( *line, "usr" ) || strstr ( *line, "sys" ) != NULL ) ) ) 
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4839,7 +4866,7 @@ int get_word_line ( int file_number, cha
             set_token_column ( file_number, *line, "runq", 0, SAR_OPTION );
             CHK_Z = _LDAVG; /* this means, skip this clause on next loop, it's enough to set set_token_column() one time */
         }
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && strstr( *line,"DEV") != NULL )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) )  && ( strstr ( *line, "DEV" ) != NULL ) )
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4848,7 +4875,7 @@ int get_word_line ( int file_number, cha
             set_token_column ( file_number, *line, "DEV", 0, SAR_OPTION );
             CHK_Z = _BLOCK_DEVICE; /* this means, skip this clause on next loop, it's enough to set set_token_column() one time */
         }
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && strstr ( *line,"rxpck/s" ) != NULL )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) ) && ( strstr ( *line,"rxpck/s" ) != NULL ) ) 
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4857,7 +4884,7 @@ int get_word_line ( int file_number, cha
             set_token_column ( file_number, *line, "rxpck", 0, SAR_OPTION );
             CHK_Z = _NETWORK; /* this means, skip this clause on next loop, it's enough to set set_token_column() one time */
         }
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && strstr ( *line,"rxerr/s" ) != NULL )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) ) && ( strstr ( *line,"rxerr/s" ) != NULL ) ) 
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4894,7 +4921,7 @@ int get_word_line ( int file_number, cha
                 }
                 CHECK_CPU_EACH++;
             }
-            else if ( strstr ( *line, "Average" ) != 0 )
+            else if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
             {
                 CHK_Z &= ~ ( 1 << 0 );  /* this means, skip this clause on next loop */
                 CHK_CORES_N = 0;
@@ -4915,7 +4942,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _TASKS )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 1 );  /* this means, skip this clause on next loop */
             else
             {
@@ -4927,7 +4954,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _PSWAP )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 2 );  /* this means, skip this clause on next loop */
             else
             {
@@ -4939,7 +4966,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _PAGING )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 3 );  /* this means, skip this clause on next loop */
             else
             {
@@ -4951,7 +4978,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _IO_TRANSFER_RATE )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 4 );  /* this means, skip this clause on next loop */
             else
             {
@@ -4963,7 +4990,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _MEMORY )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 5 );  /* this means, skip this clause on next loop */
             else
             {
@@ -4975,7 +5002,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _SWPUSED )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 6 );  /* this means, skip this clause on next loop */
             else
             {
@@ -4987,7 +5014,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _KERNEL_TABLE )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 7 );  /* this means, skip this clause on next loop */
             else
             {
@@ -4999,7 +5026,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _LDAVG )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 8 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5014,7 +5041,7 @@ int get_word_line ( int file_number, cha
          */ 
         else if ( CHK_Z == _BLOCK_DEVICE )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 9 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5029,7 +5056,7 @@ int get_word_line ( int file_number, cha
          */ 
         else if ( CHK_Z == _NETWORK )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 10 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5041,7 +5068,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _NETWORK_ERROR )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 11 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5066,7 +5093,8 @@ int get_word_line ( int file_number, cha
             && strstr ( *line, "rcvin/s" ) == 0 && strstr ( *line, "cswch/s" ) == 0 
             && strstr ( *line, "frmpg/s" ) == 0 && strstr ( *line, "call/s" ) == 0 
             && strstr ( *line, "scall/s" ) == 0 && strstr ( *line, "totsck" ) == 0 
-            && strstr ( *line, "total/s" ) == 0 && strstr ( *line, "Average" ) == 0 )
+            && strstr ( *line, "total/s" ) == 0 && strstr ( *line, "Average" ) == 0
+            && strstr ( *line, "平均" ) == 0 )
         {
             /* no match */
             return ( 0 );
@@ -5422,7 +5450,7 @@ int get_word_line ( int file_number, cha
              * cpu and sets core number every time. core number is incremented, so, lately, core number could be known (including all line)  
              * stop calling this clause if reached up to MAX_CORE_NUMBERS, anyway, it will no be called very soon   
              */
-            if ( strstr ( *line, "Average" ) &&  !strstr ( *line, "CPU" ) && ( SHOW_AVG == 2 ) && ( CHK_CORES_N <= MAX_CORE_NUMBERS ) )
+            if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "CPU" ) && ( SHOW_AVG == 2 ) && ( CHK_CORES_N <= MAX_CORE_NUMBERS ) )
             {
                 append_list ( &line_obj, str_tmp );
                 set_token_items ( file_number, line, "cpu" , CHK_CORES_N, SAR_OPTION ); /* setting each cpu values to the object */
@@ -5434,7 +5462,7 @@ int get_word_line ( int file_number, cha
             /* for devices (DEV) setting values to arrays
              * just because not knowing which block device will appear every time this clause is called, setting each name to the object for future use 
              */ 
-            else if ( strstr ( *line, "Average" )  &&  !strstr ( *line, "DEV" ) && ( SHOW_AVG == 3 ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "DEV" ) && ( SHOW_AVG == 3 ) )
             {
                 if ( strstr ( *line, "dev") != 0 )
                 {
@@ -5445,12 +5473,12 @@ int get_word_line ( int file_number, cha
             /* for devices (NET rxpck/s) setting values to arrays
              * just because not knowing which network device will appear every time this clause is called, setting each name to the object for future use 
              */ 
-            else if ( strstr ( *line, "Average" )  &&  !strstr ( *line, "rxpck" ) && ( SHOW_AVG == 4 ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "rxpck" ) && ( SHOW_AVG == 4 ) )
             {
                 append_list ( &line_obj, str_tmp );
                 set_token_items ( file_number, line, "rxpck" , -999, SAR_OPTION );
             }
-            else if ( strstr ( *line, "Average" )  &&  !strstr ( *line, "rxerr" ) && ( SHOW_AVG == 5 ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "rxerr" ) && ( SHOW_AVG == 5 ) )
             {
                 append_list ( &line_obj, str_tmp );
                 set_token_items ( file_number, line, "rxerr" , -999, SAR_OPTION );
@@ -5459,7 +5487,7 @@ int get_word_line ( int file_number, cha
             /* Call set_token_items() here
              * this clause is for others which only have one line to be analyzed 
              */
-            else if ( strstr ( *line, "Average" )  && ( ( SHOW_AVG == 0 ) ) && ( strstr ( *line, "CPU" ) == NULL ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) && ( ( SHOW_AVG == 0 ) ) && ( strstr ( *line, "CPU" ) == NULL ) )
             {
                 append_list ( &line_obj, str_tmp );
                 if ( ( ( CHK_A >> 1 ) & 1 ) )
