diff -ruNp a/common.c b/common.c
--- a/common.c	2019-02-08 07:14:47.153233928 +0900
+++ b/common.c	2019-02-08 07:09:35.921003700 +0900
@@ -391,7 +391,7 @@ void read_sar_cpu_as_paragraph ( const c
         for ( i--; ( i > 0 ) && isspace ( line [ i -1 ] ) ; i-- )
             line [ i -1 ] = '\0';
         /* count cpu all line and set numbers to cpu_as_paragraph */
-        if ( strstr ( *(&line), "Average" ) != 0 )
+        if ( ( strstr ( *(&line), "Average" ) != 0 ) || ( strstr ( *(&line), "平均値" ) != 0 ) ) 
             break;
         if ( ( strstr ( *(&line), "all" ) != 0 ) ) 
             _cpu_as_paragraph++;
@@ -756,37 +756,37 @@ void sanity_check_file ( char * line, co
     if ( strstr ( line, "LINUX RESTART" ) != 0 )
         initialize_avg_checked_int ( );
 
-    if ( ( cpu == 1 ) && ( proc_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( cpu == 1 ) && ( proc_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         cpu_avg_checked_int = 1;
         cpu_avg_checked = "Checked";
     }
-    if ( ( proc_s == 1 ) && ( pswpin_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( proc_s == 1 ) && ( pswpin_s == 0 ) && ( ( !strstr ( line, "Average" ) ) || ( !strstr ( line, "平均値" ) ) ) )
     {
         proc_s_avg_checked_int = 1;
         proc_s_avg_checked = "Checked";
     }
-    if ( ( pswpin_s == 1 ) && ( pgpgin_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( pswpin_s == 1 ) && ( pgpgin_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         pswpin_s_avg_checked_int = 1;
         pswpin_s_avg_checked = "Checked";
     }
-    if ( ( pgpgin_s == 1 ) && ( tps == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( pgpgin_s == 1 ) && ( tps == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         pgpgin_s_avg_checked_int = 1;
         pgpgin_s_avg_checked = "Checked";
     }
-    if ( ( tps == 1 ) && ( kbmemfree == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( tps == 1 ) && ( kbmemfree == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         tps_avg_checked_int = 1;
         tps_avg_checked = "Checked";
     }
-    if ( ( kbmemfree == 1 ) && ( kbswpfree == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( kbmemfree == 1 ) && ( kbswpfree == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         kbmemfree_avg_checked_int = 1;
         kbmemfree_avg_checked = "Checked";
     }
-    if ( ( kbswpfree == 1 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( kbswpfree == 1 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         if ( ( kbhugfree == 1 ) || ( dentunusd == 0 ) )
         {
@@ -796,47 +796,47 @@ void sanity_check_file ( char * line, co
             kbhugfree_avg_checked = "Checked";
         }
     }
-    if ( ( kbhugfree == 1 ) && ( dentunusd == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( kbhugfree == 1 ) && ( dentunusd == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         kbhugfree_avg_checked_int = 1;
         kbhugfree_avg_checked = "Checked";
     }
-    if ( ( dentunusd == 1 ) && ( runq_sz == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( dentunusd == 1 ) && ( runq_sz == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         dentunusd_avg_checked_int = 1;
         dentunusd_avg_checked = "Checked";
     }
-    if ( ( runq_sz == 1 ) && ( dev == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( runq_sz == 1 ) && ( dev == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         runq_sz_avg_checked_int = 1;
         runq_sz_avg_checked = "Checked";
     }
-    if ( ( dev == 1 ) && ( rxpck_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( dev == 1 ) && ( rxpck_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         dev_avg_checked_int = 1;
         dev_avg_checked = "Checked";
     }
-    if ( ( rxpck_s == 1 ) && ( rxerr_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( rxpck_s == 1 ) && ( rxerr_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         rxpck_s_avg_checked_int = 1;
         rxpck_s_avg_checked = "Checked";
     }
-    if ( ( rxerr_s == 1 ) && ( call_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( rxerr_s == 1 ) && ( call_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         rxerr_s_avg_checked_int = 1;
         rxerr_s_avg_checked = "Checked";
     }
-    if ( ( call_s == 1 ) && ( scall_s == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( call_s == 1 ) && ( scall_s == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         call_s_avg_checked_int = 1;
         call_s_avg_checked = "Checked";
     }
-    if ( ( scall_s == 1 ) && ( totsck == 0 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( scall_s == 1 ) && ( totsck == 0 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         scall_s_avg_checked_int = 1;
         scall_s_avg_checked = "Checked";
     }
-    if ( ( totsck == 1 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( totsck == 1 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         if ( squeezd == 0 )
         {
@@ -846,7 +846,7 @@ void sanity_check_file ( char * line, co
             squeezd_avg_checked = "Checked";
         }
     }
-    if ( ( squeezd == 1 ) && ( strstr ( line, "Average" ) != 0 ) )
+    if ( ( squeezd == 1 ) && ( ( strstr ( line, "Average" ) != 0 ) || ( strstr ( line, "平均値" ) != 0 ) ) )
     {
         squeezd_avg_checked_int = 1;
         squeezd_avg_checked = "Checked";
diff -ruNp a/sar_analyzer.c b/sar_analyzer.c
--- a/sar_analyzer.c	2019-02-08 07:30:43.334476872 +0900
+++ b/sar_analyzer.c	2019-02-08 07:10:04.911466263 +0900
@@ -266,7 +266,7 @@ int set_token_column ( int file_number,
         /* if AM or PM will appear, column number should be decremented, unless, segfault in set_column_item() */
         if ( SAR_OPTION != 'Z' )
         {
-            if ( ( strstr ( token, "AM" ) != NULL ) || ( strstr ( token, "PM" ) != NULL ) )
+            if ( ( strstr ( token, "AM" ) != NULL ) || ( strstr ( token, "PM" ) != NULL ) || ( strstr ( token, "午前" ) != NULL ) || ( strstr ( token, "午後" ) != NULL ) ) 
                 i--;
         }
         /* this item name is set by get_word_line() */
@@ -783,12 +783,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_usr [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -903,12 +904,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_sys [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -1008,12 +1010,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_iowait [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -1113,12 +1116,13 @@ int set_token_items ( int file_number, c
                     /* code for postscript file utility 0 means, CPU all */
                     if ( utility == 0 )
                     {
-                        if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                           ( strstr ( time_value, ":10:" ) != NULL ) ||
-                           ( strstr ( time_value, ":20:" ) != NULL ) ||
-                           ( strstr ( time_value, ":30:" ) != NULL ) ||
-                           ( strstr ( time_value, ":40:" ) != NULL ) ||
-                           ( strstr ( time_value, ":50:" ) != NULL ) )
+                        if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                           ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                           ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                           ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                           ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                           ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                        )
                         {
                             horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_cpu_idle [ file_number ] );
                             vertical_value = vertical_start_point_cpu_all + ( vertical_notch_percent * t );
@@ -1215,12 +1219,13 @@ int set_token_items ( int file_number, c
                         set_tasks_lowest_date ( this_date_all, "proc" );
                         set_tasks_lowest_time ( time_value, "proc" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_tasks_proc [ file_number ] );
@@ -1311,12 +1316,13 @@ int set_token_items ( int file_number, c
                         set_tasks_lowest_date ( this_date_all, "cswch" );
                         set_tasks_lowest_time ( time_value, "cswch" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_tasks_cswch [ file_number ] );
@@ -1413,12 +1419,13 @@ int set_token_items ( int file_number, c
                         set_pswap_lowest_date ( this_date_all, "pswpin" );
                         set_pswap_lowest_time ( time_value, "pswpin" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_swapping_pswpin [ file_number ] );
@@ -1509,12 +1516,13 @@ int set_token_items ( int file_number, c
                         set_pswap_lowest_date ( this_date_all, "pswpout" );
                         set_pswap_lowest_time ( time_value, "pswpout" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_swapping_pswpout [ file_number ] );
@@ -1632,12 +1640,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "pgpgin" );
                         set_paging_lowest_time ( time_value, "pgpgin" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_pgpgin [ file_number ] );
@@ -1728,12 +1737,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "pgpgout" );
                         set_paging_lowest_time ( time_value, "pgpgout" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_pgpgout [ file_number ] );
@@ -1812,12 +1822,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "fault" );
                         set_paging_lowest_time ( time_value, "fault" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_fault [ file_number ] );
@@ -1895,12 +1906,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "majflt" );
                         set_paging_lowest_time ( time_value, "majflt" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_mjflt [ file_number ] );
@@ -1978,12 +1990,13 @@ int set_token_items ( int file_number, c
                         set_paging_lowest_date ( this_date_all, "vmeff" );
                         set_paging_lowest_time ( time_value, "vmeff" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_paging_vmeff [ file_number ] );
@@ -2088,12 +2101,13 @@ int set_token_items ( int file_number, c
                         set_io_transfer_rate_lowest_date ( this_date_all, "tps" );
                         set_io_transfer_rate_lowest_time ( time_value, "tps" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_io_transfer_rate_tps [ file_number ] );
@@ -2184,12 +2198,13 @@ int set_token_items ( int file_number, c
                         set_io_transfer_rate_lowest_date ( this_date_all, "bread" );
                         set_io_transfer_rate_lowest_time ( time_value, "bread" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_io_transfer_rate_bread [ file_number ] );
@@ -2267,12 +2282,13 @@ int set_token_items ( int file_number, c
                         set_io_transfer_rate_lowest_date ( this_date_all, "bwrtn" );
                         set_io_transfer_rate_lowest_time ( time_value, "bwrtn" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_io_transfer_rate_bwrtn [ file_number ] );
@@ -2377,12 +2393,13 @@ int set_token_items ( int file_number, c
                         set_memory_lowest_date ( this_date_all, "memused" );
                         set_memory_lowest_time ( time_value, "memused" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_memory_memused [ file_number ] );
@@ -2473,12 +2490,13 @@ int set_token_items ( int file_number, c
                         set_memory_lowest_date ( this_date_all, "kbcommit" );
                         set_memory_lowest_time ( time_value, "kbcommit" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_memory_kbcommit [ file_number ] );
@@ -2556,12 +2574,13 @@ int set_token_items ( int file_number, c
                         set_memory_lowest_date ( this_date_all, "commit" );
                         set_memory_lowest_time ( time_value, "commit" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_memory_commit [ file_number ] );
@@ -2741,12 +2760,13 @@ int set_token_items ( int file_number, c
                         set_kernel_table_lowest_date ( this_date_all, "dentunusd" );
                         set_kernel_table_lowest_time ( time_value, "dentunusd" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_kernel_table_dentunusd [ file_number ] );
@@ -2837,12 +2857,13 @@ int set_token_items ( int file_number, c
                         set_kernel_table_lowest_date ( this_date_all, "file" );
                         set_kernel_table_lowest_time ( time_value, "file" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_kernel_table_file [ file_number ] );
@@ -2920,12 +2941,13 @@ int set_token_items ( int file_number, c
                         set_kernel_table_lowest_date ( this_date_all, "inode" );
                         set_kernel_table_lowest_time ( time_value, "inode" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_kernel_table_inode [ file_number ] );
@@ -3045,12 +3067,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "runq_sz" );
                         set_ldavg_lowest_time ( time_value, "runq_sz" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_runq [ file_number ] );
@@ -3142,12 +3165,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "plist_sz" );
                         set_ldavg_lowest_time ( time_value, "plist_az" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_plist [ file_number ] );
@@ -3226,12 +3250,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "ldavg_one" );
                         set_ldavg_lowest_time ( time_value, "ldavg_one" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_ldavg_one [ file_number ] );
@@ -3309,12 +3334,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date( this_date_all, "ldavg_five" );
                         set_ldavg_lowest_time( time_value, "ldavg_five" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_ldavg_five [ file_number ] );
@@ -3392,12 +3418,13 @@ int set_token_items ( int file_number, c
                         set_ldavg_lowest_date ( this_date_all, "ldavg_15" );
                         set_ldavg_lowest_time ( time_value, "ldavg_15" );
                     }
-                    if ( ( strstr ( time_value, ":00:" ) != NULL ) ||
-                       ( strstr ( time_value, ":10:" ) != NULL ) ||
-                       ( strstr ( time_value, ":20:" ) != NULL ) ||
-                       ( strstr ( time_value, ":30:" ) != NULL ) ||
-                       ( strstr ( time_value, ":40:" ) != NULL ) ||
-                       ( strstr ( time_value, ":50:" ) != NULL ) )
+                    if ( ( strstr ( time_value, ":00:" ) != NULL ) || ( strstr ( time_value, "00分" ) != NULL ) ||
+                       ( strstr ( time_value, ":10:" ) != NULL ) || ( strstr ( time_value, "10分" ) != NULL ) ||
+                       ( strstr ( time_value, ":20:" ) != NULL ) || ( strstr ( time_value, "20分" ) != NULL ) ||
+                       ( strstr ( time_value, ":30:" ) != NULL ) || ( strstr ( time_value, "30分" ) != NULL ) ||
+                       ( strstr ( time_value, ":40:" ) != NULL ) || ( strstr ( time_value, "40分" ) != NULL ) ||
+                       ( strstr ( time_value, ":50:" ) != NULL ) || ( strstr ( time_value, "50分" ) != NULL )
+                    )
                     {
                         /* code for postscript file */
                         horizontal_value = horizontal_start_point + ( horizontal_notch * horizontal_value_ldavg_ldavg_15 [ file_number ] );
@@ -4851,7 +4878,7 @@ int get_word_line ( int file_number, cha
             initialize_check_int ( );
         }
         /* first, we are setting column number of interests */
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && (strstr ( *line, "usr" ) || strstr ( *line, "sys" )   != NULL ) )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) )  && ( ( strstr ( *line, "usr" ) || strstr ( *line, "sys" ) != NULL ) ) ) 
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4933,7 +4960,7 @@ int get_word_line ( int file_number, cha
             set_token_column ( file_number, *line, "runq", 0, SAR_OPTION );
             CHK_Z = _LDAVG; /* this means, skip this clause on next loop, it's enough to set set_token_column() one time */
         }
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && strstr( *line,"DEV") != NULL )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) )  && ( strstr ( *line, "DEV" ) != NULL ) )
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4942,7 +4969,7 @@ int get_word_line ( int file_number, cha
             set_token_column ( file_number, *line, "DEV", 0, SAR_OPTION );
             CHK_Z = _BLOCK_DEVICE; /* this means, skip this clause on next loop, it's enough to set set_token_column() one time */
         }
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && strstr ( *line,"rxpck/s" ) != NULL )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) ) && ( strstr ( *line,"rxpck/s" ) != NULL ) ) 
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4951,7 +4978,7 @@ int get_word_line ( int file_number, cha
             set_token_column ( file_number, *line, "rxpck", 0, SAR_OPTION );
             CHK_Z = _NETWORK; /* this means, skip this clause on next loop, it's enough to set set_token_column() one time */
         }
-        else if ( ( strstr ( *line, "Average" ) ) == 0 && strstr ( *line,"rxerr/s" ) != NULL )
+        else if ( ( ( strstr ( *line, "Average" ) == 0 ) || ( strstr ( *line, "平均値" ) == 0 ) ) && ( strstr ( *line,"rxerr/s" ) != NULL ) ) 
         {
             if ( MESSAGE_ONLY == 0 )
                 append_list ( &line_all_obj, str_tmp );
@@ -4988,7 +5015,7 @@ int get_word_line ( int file_number, cha
                 }
                 CHECK_CPU_EACH++;
             }
-            else if ( strstr ( *line, "Average" ) != 0 )
+            else if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
             {
                 CHK_Z &= ~ ( 1 << 0 );  /* this means, skip this clause on next loop */
                 CHK_CORES_N = 0;
@@ -5009,7 +5036,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _TASKS )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 1 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5021,7 +5048,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _PSWAP )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 2 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5033,7 +5060,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _PAGING )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 3 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5045,7 +5072,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _IO_TRANSFER_RATE )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 4 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5057,7 +5084,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _MEMORY )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 5 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5069,7 +5096,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _SWPUSED )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 6 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5081,7 +5108,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _KERNEL_TABLE )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 7 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5093,7 +5120,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _LDAVG )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 8 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5108,7 +5135,7 @@ int get_word_line ( int file_number, cha
          */ 
         else if ( CHK_Z == _BLOCK_DEVICE )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 9 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5123,7 +5150,7 @@ int get_word_line ( int file_number, cha
          */ 
         else if ( CHK_Z == _NETWORK )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 10 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5135,7 +5162,7 @@ int get_word_line ( int file_number, cha
         }
         else if ( CHK_Z == _NETWORK_ERROR )
         {
-            if ( strstr ( *line, "Average" ) != 0 )
+            if ( ( strstr ( *line, "Average" ) != 0 ) || ( strstr ( *line, "平均値" ) != 0 ) ) 
                 CHK_Z &= ~ ( 1 << 11 );  /* this means, skip this clause on next loop */
             else
             {
@@ -5160,7 +5187,8 @@ int get_word_line ( int file_number, cha
             && strstr ( *line, "rcvin/s" ) == 0 && strstr ( *line, "cswch/s" ) == 0 
             && strstr ( *line, "frmpg/s" ) == 0 && strstr ( *line, "call/s" ) == 0 
             && strstr ( *line, "scall/s" ) == 0 && strstr ( *line, "totsck" ) == 0 
-            && strstr ( *line, "total/s" ) == 0 && strstr ( *line, "Average" ) == 0 )
+            && strstr ( *line, "total/s" ) == 0 && strstr ( *line, "Average" ) == 0
+            && strstr ( *line, "平均" ) == 0 )
         {
             /* no match */
             return ( 0 );
@@ -5516,7 +5544,7 @@ int get_word_line ( int file_number, cha
              * cpu and sets core number every time. core number is incremented, so, lately, core number could be known (including all line)  
              * stop calling this clause if reached up to MAX_CORE_NUMBERS, anyway, it will no be called very soon   
              */
-            if ( strstr ( *line, "Average" ) &&  !strstr ( *line, "CPU" ) && ( SHOW_AVG == 2 ) && ( CHK_CORES_N <= MAX_CORE_NUMBERS ) )
+            if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "CPU" ) && ( SHOW_AVG == 2 ) && ( CHK_CORES_N <= MAX_CORE_NUMBERS ) )
             {
                 append_list ( &line_obj, str_tmp );
                 set_token_items ( file_number, line, "cpu" , CHK_CORES_N, SAR_OPTION, time_span ); /* setting each cpu values to the object */
@@ -5528,7 +5556,7 @@ int get_word_line ( int file_number, cha
             /* for devices (DEV) setting values to arrays
              * just because not knowing which block device will appear every time this clause is called, setting each name to the object for future use 
              */ 
-            else if ( strstr ( *line, "Average" )  &&  !strstr ( *line, "DEV" ) && ( SHOW_AVG == 3 ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "DEV" ) && ( SHOW_AVG == 3 ) )
             {
                 if ( strstr ( *line, "dev") != 0 )
                 {
@@ -5539,12 +5567,12 @@ int get_word_line ( int file_number, cha
             /* for devices (NET rxpck/s) setting values to arrays
              * just because not knowing which network device will appear every time this clause is called, setting each name to the object for future use 
              */ 
-            else if ( strstr ( *line, "Average" )  &&  !strstr ( *line, "rxpck" ) && ( SHOW_AVG == 4 ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "rxpck" ) && ( SHOW_AVG == 4 ) )
             {
                 append_list ( &line_obj, str_tmp );
                 set_token_items ( file_number, line, "rxpck" , -999, SAR_OPTION, time_span );
             }
-            else if ( strstr ( *line, "Average" )  &&  !strstr ( *line, "rxerr" ) && ( SHOW_AVG == 5 ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) &&  !strstr ( *line, "rxerr" ) && ( SHOW_AVG == 5 ) )
             {
                 append_list ( &line_obj, str_tmp );
                 set_token_items ( file_number, line, "rxerr" , -999, SAR_OPTION, time_span );
@@ -5553,7 +5581,7 @@ int get_word_line ( int file_number, cha
             /* Call set_token_items() here
              * this clause is for others which only have one line to be analyzed 
              */
-            else if ( strstr ( *line, "Average" )  && ( ( SHOW_AVG == 0 ) ) && ( strstr ( *line, "CPU" ) == NULL ) )
+            else if ( ( ( strstr ( *line, "Average" ) ) || ( strstr ( *line, "平均値" ) ) ) && ( ( SHOW_AVG == 0 ) ) && ( strstr ( *line, "CPU" ) == NULL ) )
             {
                 append_list ( &line_obj, str_tmp );
                 if ( ( ( CHK_A >> 1 ) & 1 ) )
